security/acme-client/src/opnsense/mvc/app/models/OPNsense/AcmeClient/AcmeClient.xml

```diff
@@ -893,6 +894,12 @@
                 <dns_hetzner_token type="TextField">
                     <Required>N</Required>
                 </dns_hetzner_token>
+                <dns_1984hosting_user type="TextField">
+                    <Required>Y</Required>
```

Comment:
Please change both to `<Required>N</Required>`, because the requirement would otherwise be applied to _all_ other DNS providers as well, effectively breaking them all.

Reply:
Done. Thanks.